src/mol-plugin-state/formats/volume.ts

```diff
@@ -197,15 +198,17 @@ export const DscifProvider = DataFormatProvider({
     parse: async (plugin, data, params?: DsCifParams) => {
         const cifCell = await plugin.build().to(data).apply(StateTransforms.Data.ParseCif).commit();
         const b = plugin.build().to(cifCell);
-        const blocks = cifCell.obj!.data.blocks.slice(1); // zero block contains query meta-data
+        const blocks = cifCell.obj!.data.blocks.slice(0); // iterate over all blocks as even 0th can contain data
 
-        if (blocks.length !== 1 && blocks.length !== 2) throw new Error('unknown number of blocks');
+        if (blocks.length !== 1 && blocks.length !== 2 && blocks.length !== 3) throw new Error('unknown number of blocks');
```

Comment:
```suggestion
        if (blocks.length === 0) throw new Error('no data blocks');
```

Now that we are checking if it is a valid volume block, we should only check if there are 0 data blocks.